src/generator/AutoRest.CSharp.Azure/Templates/AzureMethodGroupTemplate.cshtml

```diff
@@ -5,42 +5,31 @@
 @EmptyLine
 namespace @Settings.Namespace
 {
-    using System;
     using System.Linq;
```

Comment:
I notice that we've still got `System.Linq` still around... what's stopping us from dropping that?